components/dashboard/src/components/BillingAccountSelector.tsx

```diff
@@ -93,8 +78,8 @@ export function BillingAccountSelector(props: { onSelected?: () => void }) {
                     {errorMessage}
                 </Alert>
             )}
-            {teamsAvailableForAttribution === undefined && <Spinner className="m-2 h-5 w-5 animate-spin" />}
-            {teamsAvailableForAttribution && (
+            {orgs.isLoading && <Spinner className="m-2 h-5 w-5 animate-spin" />}
+            {orgs?.data && (
```

Comment:
Shouldn't need the `?` here, `orgs` will always be an object as it's the return of the `useQuery()` hook.